source/renderer/app/components/staking/widgets/TooltipPool.js

```diff
@@ -625,13 +737,35 @@ export default class TooltipPool extends Component<Props, State> {
                 {`${formattedWalletAmount(cost, true, false)}`}
               </span>
             </dd>
-            <dt>{intl.formatMessage(messages.producedBlocks)}</dt>
+            <dt>
+              <div className={styles.fieldLabel}>
+                {intl.formatMessage(messages.producedBlocks)}
+              </div>
+              <Tooltip
+                key="producedBlocks"
+                skin={TooltipSkin}
+                tip={intl.formatMessage(messages.producedBlocksTooltip)}
+              >
+                <div className={styles.questionMark}>?</div>
+              </Tooltip>
+            </dt>
             <dd className={styles.defaultColor}>
               <span className={styles.defaultColorContent}>
                 {shortNumber(producedBlocks)}
               </span>
             </dd>
-            <dt>{intl.formatMessage(messages.potentialRewards)}</dt>
+            <dt>
```

Comment:
@yakovkaravelov Since we are repeating a lot of these similar sections wouldn't be a better implementation to have instead:

`<dt>
              <div className={styles.fieldLabel}>
                {intl.formatMessage(messages.producedBlocks)}
              </div>
              <Tooltip
                key="producedBlocks"
                skin={TooltipSkin}
                tip={intl.formatMessage(messages.producedBlocksTooltip)}
              >
                <div className={styles.questionMark}>?</div>
              </Tooltip>
  </dt>`

some method which will render every section with given params

renderTooltipField(message: string, tooltipMessage: string, key: string, hasQuestionMark: boolean)

Reply:
@DeeJayElly Each element has different css classes and also structure is a bit different from element to element.
So it will make additional complexity.

Reply:
@DeeJayElly I refactored to make this more simpler.

Reply:
Cool this is better, gj @yakovkaravelov .